### DISCUSSION TITLE
React for SSR & preact just in the browser?

### DISCUSSION THREAD

**Discussion by kurtextrem** [preactjs/preact#3537] [General]
Has anyone tried using React for SSR (e.g. v18) & preact just in the browser (so only the hydration + adding event listeners)?

I'm not sure if this frankenstein approach is feasible, but since preact is smaller and faster in hydrating, this might be worth exploring, while having 'full' compatibility with React 18 on the server-side. This would mean for server code, webpack or other bundlers would not alias react-dom/react and for the client bundle it would.

**Comment by developit** [comment]
This should work, but it's not something we officially support.
